sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -166,6 +170,21 @@ export interface KeyVaultSelectiveKeyRestoreResult {
     startTime: Date;
 }
 
+// @public
+export interface KeyVaultSetting {
+    name: string;
+    value: boolean;
+}
+
+// @public
+export class KeyVaultSettingsClient {
+    constructor(vaultUrl: string, credential: TokenCredential, options?: SettingsClientOptions);
+    getSetting(settingName: string, options?: GetSettingOptions): Promise<KeyVaultSetting>;
```

Comment:
should this return the boolean value directly since you are passing in the name?

Reply:
Think this comment has been more or less resolved with the change in approach and our offline discussion but feel free to unresolve if not :)